pylibui/controls/__init__.py

```diff
@@ -4,6 +4,10 @@
 """
 
 from .box import Box, HorizontalBox, VerticalBox
+from .button import Button
 from .control import Control
+from .checkbox import Checkbox
 from .label import Label
-from .window import Window
+from .progressbar import ProgressBar
+from .slider import Slider
+from .window import Window
```

Comment:
It's important to add a newline after the last piece of code. Some IDE's can be set to include it automatically.